calc/src/calc_payout.rs

```diff
@@ -102,3 +122,130 @@ impl CalcPayout {
         own_staking_payout.to_string()
     }
 }
+
+#[cfg(test)]
+mod test_payout {
+    use super::*;
+    /*
+    These tests are designed to test the accuracy of the preceding functions.
+    All the data can be retrieved and checked with polkadot-js and subscan.io.
+    NOTE: As these tests are setup, the eras should be regularly updated.
```

Comment:
Im a bit confused by the `NOTE`. Does this mean these tests should be updated in the future?

Reply:
Noting this was a mistake on my part, only added confusion. It originated due to polkadot-js retrieveng upt to a certain amount of past eras, so I added the block number to the data so that it's easier to check the values and removed this `NOTE`.

Reply:
Ahh makes sense!

Thanks for the explanation